unload/base.py

```diff
@@ -125,63 +169,121 @@ def add_module(modules, module, line_number):
+            :member: the member's name
+            :line_number: the line number at which the member is loaded
+            :returns: a modified members dictionary
             """
-            if tag not in tags:
-                tags[tag] = [line_number]
+            if member not in members:
+                members[member] = [line_number]
             else:
-                # The same tag can be loaded multiple times
-                if line_number not in tags[tag]:
-                    tags[tag].append(line_number)
+                # The same member can be loaded multiple times
+                if line_number not in members[member]:
+                    members[member].append(line_number)
 
-            return tags
+            return members
 
         modules = {}
-        tags = {}
+        members = {}
 
         for token in self.tokens:
             token_content = token.split_contents()
             # Extract load blocks
             if token.token_type == 2 and token_content[0] == 'load':
-                # FROM syntax is used; individual tags are loaded
+                # FROM syntax is used; individual members are loaded
                 if token_content >= 4 and token_content[-2] == 'from':
                     # Add loaded module
                     module = token_content[-1]
                     modules = add_module(modules, module, token.lineno)
-                    # Add loaded tags
-                    loaded_tags = token_content[1:-2]
-                    for tag in loaded_tags:
-                        tags = add_tag(tags, tag, token.lineno)
+                    # Add loaded members
+                    loaded_members = token_content[1:-2]
+                    for member in loaded_members:
+                        members = add_member(members, member, token.lineno)
                 # Regular syntax
                 else:
                     # Multiple modules can be imported in the same load block
                     templatetags_modules = token_content[1:]
                     for module in templatetags_modules:
                         modules = add_module(modules, module, token.lineno)
 
-        return modules, tags
+        return modules, members
 
-    def _get_used_templatetags(self):
+    def _get_used_tags(self):
         """
         Get the list of custom template tags used in the template.
 
-        :returns: a list of custom template tags
+        :returns: a list of custom tag names
         """
         used_tags = []
         for token in self.tokens:
             token_content = token.split_contents()
             # Extract blocks that do not contain one of the built-in tags
             if (token.token_type == 2 and
                     token_content[0] not in BUILT_IN_TAGS.keys() and
-                    token_content[0] not in set(BUILT_IN_TAGS.values())):
+                    # Skip built-in 'end' tags
+                    token_content[0] not in set(BUILT_IN_TAGS.values()) and
```

Comment:
couldn't you perhaps join all this beforehand and then do a single check? `token_content[0] not in joined_stuff`